### DISCUSSION TITLE
Renew and State field questions

### DISCUSSION THREAD

**Discussion by SooshieSky** [etsy/open-api#978] [Q&A]
I have a listing that's "expired" in that **ending_timestamp** is 21 hours in the past, yet the listing still returns the status of **active**.  I'm trying to renew this listing, but the **renew** field from v2's **updateListing** is missing from v3.  I've tried toggling **should_auto_renew** on and off to see if that would renew the listing, but it didn't work.  What is the process for manually renewing a listing in v3?  And is it normal that an "expired" listing would still be showing in the **active** state?

**Comment by Gareth064** [comment]
I have just used getListing by ID for one of my expired listings and it shows a state of "expired".
If you're listing is definitely showing as expired on the GUI but not in the API results then I recommend you email developer@etsy.com to ask them to investigate.
GUI
![image](https://user-images.githubusercontent.com/22307223/234375709-220fa959-dc42-442f-ba78-043139bbf136.png)
API
![image](https://user-images.githubusercontent.com/22307223/234375213-8e4b2010-9aff-4900-b502-764332069870.png)
